docs/get_started/index.md

```diff
@@ -11,141 +11,70 @@ imperative operations on the fly.  A graph optimization layer on top of that
 makes symbolic execution fast and memory efficient. The MXNet library is
```

Comment:
1. "Test MXNet installation" and "Common Installation Issue or Troubleshooting" are 2 sections that will be useful. Current "overview" section is serving purpose of "Test MXNet Installation".
2. In place of "Overview" we should probably have more better example as this is a getting started page.
3. Should we move the installation to separate page and link it from getting started page?
4. Programming languages in the middle of the page is somewhat not very good looking.